common/persistence/dataInterfaces.go

```diff
@@ -183,6 +185,8 @@ type (
 		GetType() int
 		GetTaskID() int64
 		SetTaskID(id int64)
+		GetVisibilityTimestamp() time.Time
```

Comment:
I don't like the name visibility timestamp. In my mind it evokes the visibility store.
How about DueTime?